src/DropdownItem.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+use InvalidArgumentException;
+use Stringable;
+use Yiisoft\Html\Html;
+use Yiisoft\Html\Tag\A;
+use Yiisoft\Html\Tag\Button;
+use Yiisoft\Html\Tag\Hr;
+use Yiisoft\Html\Tag\Li;
+use Yiisoft\Html\Tag\Span;
+
+final class DropdownItem
+{
+    private const DROPDOWN_DIVIDER_CLASS = 'dropdown-divider';
+    private const DROPDOWN_ITEM_ACTIVE_CLASS = 'active';
+    private const DROPDOWN_ITEM_CLASS = 'dropdown-item';
+    private const DROPDOWN_ITEM_DISABLED_CLASS = 'disabled';
+    private const DROPDOWN_ITEM_HEADER_CLASS = 'dropdown-header';
+    private const DROPDOWN_ITEM_TEXT_CLASS = 'dropdown-item-text';
+    private Li|null $content = null;
```

Comment:
Better create LI tag in `Dropdown` class and use `string|`Stringable`|null` type here.

Reply:
This is the helper to create the dropdowns and the items inside its structure return a Li tag, so it is correct.

Reply:
ok